applications/vessel-history/src/features/profile/components/Event.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+// import { Link } from 'react-router-dom'
+import cx from 'classnames'
+import dayjs from 'dayjs'
```

Comment:
Let's talk about date format libraries in next frontend meeting to align on using the same across projects

Reply:
As a base to discuss on the next meeting, here's our experience with date/time libraries so far:
- moment.js is too big, unless stripping some of its codebase, and its own authors mention it as deprecated
- date-fns is lightweight but syntax is awkward
- dayjs has a nice, lean, moment-like API, but it's **UTC time/timezones capabilities**, which are crucial in GFW, are not good/reliable enough (or were when we checked)
- we settled for now on <a href="https://moment.github.io/luxon/">Luxon</a>, from moment creators:
  - it has an clear and explicit API - no magic formats like 'LLLL', more like DateTime.DATE_MED
  - it handles UTC time well
  - it is lightweight 
  - it provides 18n capabilities by relying on native browser i18n capabilities
  
Let's discuss that, also taking the opportunity to remind us that it's important to **always work with UTC times** in GFW projects. We had a lot of errors caused by not paying attention to that (although in Argentina if you use local time by mistake the problem should more obvious, but for us we are just 1hr away from UTC time)  

Example: 
https://github.com/GlobalFishingWatch/frontend/blob/f5519e6b8eb57201efd792c4fe4af674f478c67b/packages/layer-composer/src/generators/heatmap/util/time-chunks.ts#L40

Reply:
Makes sense, let's use luxon from now on, this code was not revisited yet so still needs to be cleaned.